cmd/references/updateref.go

```diff
@@ -49,7 +49,7 @@ func init() {
 		"", "Description for the reference")
 	UpdateCmd.Flags().StringVarP(&resourceType, "restype", "s",
 		"", "Resource type must be KeyStore or TrustStore")
-	UpdateCmd.Flags().StringVarP(&refers, "refers", "r",
+	UpdateCmd.Flags().StringVarP(&refers, "refers", "",
```

Comment:
why was this removed? Is it because of the global `r` flag? If yes, then removing the existing `r` for `refers` is breaking change right?